### ISSUE TITLE
Rolling Reload Scattershot, and Replenishment does no reset Arcane Barrage and Scattershot

### ISSUE THREAD

**Issue by ghost** [KKonaW/gunner-script#1]
Unless im installing the script wrongly, RR and replenishment doesn't seem to reset cd of these 2 skills as intended. Other stuff work great, thanks for the script.

**Comment by ChicknWing** [comment]
Yeah and also I can't do scattershot after 2nd roll for some reason. I press the skill and nothing happens.

**Comment by SERVANTSKILL** [comment]
I've noticed this as well but unfortunately I guess since this is a leaked script chances are it's probably not going to be fixed :(

**Comment by unstablepenis** [comment]
how do i use this?
